tests/gtest/aviflosslesstest.cc

```diff
@@ -36,7 +40,18 @@ TEST(BasicTest, EncodeDecodeMatrixCoefficients) {
           /*ignoreXMP=*/false, image.get(),
           /*outDepth=*/nullptr, /*sourceTiming=*/nullptr,
           /*frameIter=*/nullptr);
-      ASSERT_NE(file_format, AVIF_APP_FILE_FORMAT_UNKNOWN);
+#if defined(AVIF_ENABLE_EXPERIMENTAL_YCGCO_R)
+      if (matrix_coefficient == AVIF_MATRIX_COEFFICIENTS_YCGCO_RO) {
+        // AVIF_MATRIX_COEFFICIENTS_YCGCO_RO does not work because the input
+        // depth is not odd.
+        ASSERT_EQ(file_format, AVIF_APP_FILE_FORMAT_UNKNOWN);
+        continue;
+      } else {
```

Comment:
Omit this `else` because it follows a `continue` statement. This will allow us t delete lines 52-54.

Reply:
Done